src/components/shared/formItems/ProjectPayoutMods.tsx

```diff
@@ -461,35 +483,59 @@ export default function ProjectPayoutMods({
               }
             />
           ) : null}
+          <Form.Item
+            label="Amount"
+            // Display message to user if the amount they inputted
+            // will result in percentage with > 2 decimal places
+            extra={
+              countDecimals(form.getFieldValue('percent')) > 2
+                ? `Will be rounded to ${form
+                    .getFieldValue('percent')
+                    .toFixed(2)}%`
+                : ''
+            }
+          >
+            <div
+              style={{
+                display: 'flex',
+                color: colors.text.primary,
+                alignItems: 'center',
+              }}
+            >
+              <div style={{ marginRight: 5 }}>
+                <CurrencySymbol currency={currency} />
```

Comment:
**suggestion**: antd has a nice way of handling this with the `addonBefore` prop on the `Input` component. 
Docs are here (look for `Pre/Post tab`): https://ant.design/components/input/